discovery-infra/test_infra/consts.py

```diff
@@ -7,7 +7,7 @@
 STORAGE_PATH = "/var/lib/libvirt/openshift-images"
 SSH_KEY = "ssh_key/key.pub"
 NODES_REGISTERED_TIMEOUT = 60 * 20
-CLUSTER_INSTALLATION_TIMEOUT = 60 * 60  # 60 minutes
+CLUSTER_INSTALLATION_TIMEOUT = 2 * 60 * 60  # 2 hours
```

Comment:
Cannot understand why it is needed, start_install_and_wait_for_installed uses 3 different method in order to declare ready:
wait_for_hosts_to_install
wait_for_operators_to_finish
wait_for_install
each method has 1 hour timeout, dont you think its enough?

Reply:
Maybe we can even add `wait_for_finalizing` just for even shorter timeouts?

Reply:
but finalizing is where the operators are installed, so we do need a long timeout